playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify Host Requirements
+  hosts: oo_all_hosts
```

Comment:
@sosiouxme shall we limit this also to run only on masters and nodes, or is it enough to filter it on the action plugin level?